src/features/Classes/EnrollStudent/index.jsx

```diff
@@ -113,12 +140,13 @@ const EnrollStudent = ({ isOpen, onClose, queryClassId }) => {
             <Form onSubmit={handleEnrollStudent}>
               <FormGroup controlId="studentInfo">
                 <Form.Control
-                  type="email"
+                  as="textarea"
```

Comment:
This new change make the distances between the text and its borders see weir, maybe a `min-heigth: 44px` could make see this more better.

![image](https://github.com/user-attachments/assets/580d81dc-4445-4a7a-9dfa-898fa4d602eb)